docs/en/config/radio.md

```diff
@@ -153,6 +153,26 @@ To map a PARAM tuning channel to a parameter:
 You can clear all parameter/tuning channel mappings by selecting menu **Tools > Clear RC to Param** at the top right of the _Parameters_ screen.
 :::
 
+## RC Input Enable/Disable Switch
+
+You can configure PX4 to use a switch channel on your transmitter to enable or disable R/C inputs to the rest of the PX4 system. This is convenient for flight testing companion software, as it allows a safety pilot to leave their R/C transmitter turned on, but without fear that bumping the sticks will override the companion software control. By default, this behavior is disabled.
+
+Set the [RC_MAP_RC_ENABLE](../advanced_config/parameter_reference.md#RC_MAP_RC_ENABLE) parameter to the channel index of the switch you want to use for this functionality. You can use [RC_ENABLESW_TH](../advanced_config/parameter_reference.md#RC_ENABLESW_TH) to configure the behavior of this switch.
+
+When the switch is asserted, R/C stick inputs and switch interactions will have no effect on the autopilot. You can test this functionality prior to arming with the following procedure:
+
+1. Assert the R/C enable switch
+2. Switch into a manual flight mode
+3. Deassert the R/C enable switch
+4. Observe in QGroundControl that manual control inputs have been lost
```

Comment:
Does this mean that the RC Loss failsafe will trigger? https://docs.px4.io/main/en/config/safety.html#manual-control-loss-failsafe - do you need to disable it?  If so, this feels flawed.

Reply:
Yes.

_As documented_, [COM_RC_LOSS_T](https://docs.px4.io/main/en/advanced_config/parameter_reference.html#COM_RC_LOSS_T) triggers if manual control lost in all modes - except those specified in [COM_RCL_EXCEPT](https://docs.px4.io/main/en/advanced_config/parameter_reference.html#COM_RCL_EXCEPT). The loss is recoverable.

We need to document that here and arguably also in the parameters themselves. Arguably you might auto set the bitmask to automatically set the mission and offboard bit - ask @MaEtUgR about his opinion on that.